hazelcast/src/main/java/com/hazelcast/spi/Operation.java

```diff
@@ -16,6 +16,7 @@
 
 package com.hazelcast.spi;
 
+import com.hazelcast.internal.cluster.ClusterClock;
```

Comment:
Just a note (and little reminder to myself): This will lead to an unused import issue in SonarQube, since `ClusterClock` is not used in the class itself, just JavaDoc. This is why we have normally have the FQN in JavaDoc. I don't know by heart from which tool this complain comes and why we have it just in SonarQube, so either we play by that rule or change it.